typings/environment.d.ts

```diff
@@ -44,5 +45,7 @@ export interface Environment extends DefaultElements<EnvironmentProps>, MetaSys<
   getLocales(): Promise<Collection<Locale>>,
   getUiExtension(id: string): Promise<UIExtension>,
   getUiExtensions(): Promise<Collection<UIExtension>>,
+  getAppInstallation(id: string): Promise<AppInstallation>,
+  getAppInstallations(): Promise<Collection<AppInstallation>>,
```

Comment:
are you also going to add createAppInstallation? It's unclear to me from Jakub's comment above

Reply:
@FLoppix

Reply:
Yes we will !